src/torchmetrics/detection/mean_ap.py

```diff
@@ -545,7 +560,7 @@ def compute(self) -> dict:
                     coco_eval.accumulate()
                     coco_eval.summarize()
                     stats = coco_eval.stats
-                    result_dict.update(self._coco_stats_to_tensor_dict(stats, prefix=prefix))
+                    result_dict.update(self._coco_stats_to_tensor_dict(stats, prefix=prefix, iou_type=self.iou_type))
```

Comment:
```suggestion
                    result_dict.update(self._coco_stats_to_tensor_dict(stats, prefix=prefix))
```
no need to pass the argument